### PR TITLE
List Structure

### PR DESCRIPTION
**Pull request by Sajeelzafar**
### In this milestone, the following tasks have been completed:

- A new directory is set up using webpack and linters.
- A layout of the to-do tasks has been created.
- The to-do list items are created dynamically when the page refreshes.
- Appropriate styling is done to resemble the original design.

### CODE REVIEW

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @Sajeelzafar, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, without a README file

- [x] Please add a professional README file to your project using the [Microverse Readme Template](https://github.com/microverseinc/readme-template)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-# to-do-list
\ No newline at end of file
+# Installing-webpack
```
Comment (BertrandConxy) [inline README.md]: - [x] As suggested by the previous reviewer, your README file is not professional as it doesn't have contents that describe your project. Microverse has provided a [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) that you should use for all your projects 😎 Please use it and customize it to your project, Make sure that all sections are covered 🏃

**Review by BertrandConxy — CHANGES_REQUESTED** [review]
Hi @Sajeelzafar, 

Good job so far! 👏 👏 
There are some issues that you still need to work on to go to the next project but you are almost there! 🏃‍♂️ 🏃‍♂️ 


## Highlights 💪 
- No linter errors
- correct flow is used
- good PR

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BertrandConxy — APPROVED** [review]
Hi @Sajeelzafar, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💪 
- No linter errors
- project requirements are met
- project works fine
- good PR description
- correct flow is used

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**